### ISSUE TITLE
Missing button "Done" when editing a mandate

### ISSUE THREAD

**Issue by MyriamVanB** [Project60/org.project60.sepa#264]
In the Edit Mandate Screen, there is no button "done" to go back to the previous screen or the contact summary screen

**Comment by systopia** [comment]
You're right, the 'edit mandate' screen does not follow the general CiviCRM logic. There is a reason for this, but it might be better for the user to change that.

However, this could amount to a lot of work, so my question is: @MyriamVanB: is this very important to you?

**Comment by systopia** [comment]
Not important right now.
